apps/web/pages/booking/direct/[...link].tsx

```diff
@@ -175,7 +175,7 @@ export default function Directlink({ booking, reason, status }: inferSSRProps<ty
                         {t(getRecipientStart("MMMM").toLowerCase())} {getRecipientStart("D, YYYY")}
                         <br />
                         {getRecipientStart("h:mma")} - {getRecipientEnd("h:mma")}{" "}
-                        <span style={{ color: "#888888" }}>({booking.attendees[0].timeZone})</span>
+                        <span style={{ color: "#888888" }}>({booking?.user?.timeZone})</span>
```

Comment:
Is this intended?

Reply:
Yes, sorry I forgot to do a self-review of this second PR copying fixes from another.
We need to show the organizer timezone here, which is available from `booking.user` as used above too.